### PR TITLE
functions/jeremydatud/js 103

### PR DESCRIPTION
**Pull request by jeremyhue1999**
# Related tickets
https://codeandtheory.atlassian.net/browse/JS-290

## Description
Adds the necessary codes for the task(functions). 

## Validation
Run HTML files
Change the url for the script attribute to see the different activities or challenges

### CODE REVIEW

File: JS-103: JavaScript Functions/challenges/js/challenge2.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function getYear() {
+  const year = new Date().getFullYear();
```
Comment (athanph) [inline JS-103: JavaScript Functions/challenges/js/challenge2.js]: We can simplify this by immediately returning the value of year :
`return new Date().getFullYear();`

File: JS-103: JavaScript Functions/challenges/js/challenge4.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function max(num1, num2) {
+  if (num1 > num2) {
```
Comment (athanph) [inline JS-103: JavaScript Functions/challenges/js/challenge4.js:2]: @jeremyhue1999 I know this is a challenge for if/else (and no need to modify)  but in practice, we can simplify this as:
```
if (num1 > num2) return true;
return false;
```

File: JS-103: JavaScript Functions/workspace/js/multiple-return.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+function isFieldEmpty() {
+  const field = document.querySelector(".info2");
+  if (!field.value) {
+    return true;
+  } else {
+    false;
```
Comment (athanph) [inline JS-103: JavaScript Functions/workspace/js/multiple-return.js]: @jeremyhue1999 The `else` block lacks the `return` keyword.

**Review by athanph — CHANGES_REQUESTED** [review]
Left some comments to address.

**Comment by jeremyhue1999** [comment]
Good morning sir, 
I changed the things that you mentioned in the comments.

**Pull request merged**